graphql/resolver.go

```diff
@@ -325,5 +325,22 @@ func New(apiURL string) Config {
 	c.Directives.RedactSecrets = func(ctx context.Context, obj any, next graphql.Resolver) (res any, err error) {
 		return next(ctx)
 	}
+	c.Directives.RequireEVGAdmin = func(ctx context.Context, obj any, next graphql.Resolver) (res any, err error) {
+		dbUser := gimlet.GetUser(ctx)
+
+		permissions := gimlet.PermissionOpts{
+			Resource:      evergreen.SuperUserPermissionsID,
+			ResourceType:  evergreen.SuperUserResourceType,
+			Permission:    evergreen.PermissionAdminSettings,
+			RequiredLevel: evergreen.AdminSettingsEdit.Value,
+		}
+		if dbUser != nil {
```

Comment:
using `mustHaveUser` from above makes it so that you don't have to check if `dbUser` is `nil`